setup.py

```diff
@@ -7,52 +7,48 @@
 
 
 def read(*parts):
-    return codecs.open(path.join(path.dirname(__file__), *parts),
-                       encoding='utf-8').read()
+    return codecs.open(
```

Comment:
given that `setup.py` is deprecated it might be worth upgrading to the new `pypackage.toml` setup

Reply:
Invoking `setup.py` directly is deprecated. Keeping configuration in `setup.py` is not deprecated. It's one of the two "official" ways to configure a package mentioned by python docs: https://packaging.python.org/en/latest/tutorials/packaging-projects/#configuring-metadata
IMHO we should keep it. But if you really want to change it, it should be another PR for readability.